outputs/quickwit.go

```diff
@@ -0,0 +1,202 @@
+}
+
+type QuickwitDocMapping struct {
+	DynamicMapping QuickwitDynamicMapping `json:"dynamic_mapping"`
+	FieldMappings  []QuickwitFieldMapping `json:"field_mappings"`
+	Mode           string                 `json:"mode"`
+	StoreSource    bool                   `json:"store_source"`
+	TimestampField string                 `json:"timestamp_field"`
+}
+
+type QuickwitMappingPayload struct {
+	Id             string                 `json:"index_id"`
+	Version        string                 `json:"version"`
+	SearchSettings QuickwitSearchSettings `json:"search_settings"`
+	DocMapping     QuickwitDocMapping     `json:"doc_mapping"`
+}
+
+func (c *Client) checkQuickwitIndexAlreadyExists(args types.InitClientArgs) bool {
+	config := args.Config.Quickwit
+
+	endpointUrl := fmt.Sprintf("%s/%s/indexes/%s/describe", config.HostPort, config.ApiEndpoint, config.Index)
+	quickwitCheckClient, err := InitClient("QuickwitCheckAlreadyExists", endpointUrl, config.MutualTLS, config.CheckCert, args)
+	if err != nil {
+		return false
+	}
+
+	if nil != quickwitCheckClient.sendRequest("GET", "") {
+		return false
+	}
+
+	return true
+}
+
+func (c *Client) AutoCreateQuickwitIndex(args types.InitClientArgs) error {
+	config := args.Config.Quickwit
+
+	if c.checkQuickwitIndexAlreadyExists(args) {
+		return nil
+	}
+
+	endpointUrl := fmt.Sprintf("%s/%s/indexes", config.HostPort, config.ApiEndpoint)
+	quickwitInitClient, err := InitClient("QuickwitInit", endpointUrl, config.MutualTLS, config.CheckCert, args)
+	if err != nil {
+		return err
+	}
+
+	mapping := &QuickwitMappingPayload{
+		Id:      config.Index,
+		Version: config.Version,
+		DocMapping: QuickwitDocMapping{
+			Mode:           "dynamic",
+			StoreSource:    true,
+			TimestampField: "time",
+			DynamicMapping: QuickwitDynamicMapping{
+				Description: "Falco",
+				Fast:        true,
+				ExpendDots:  true,
+				Indexed:     true,
+				Stored:      true,
+				Record:      "basic",
+				Tokenizer:   "raw",
+			},
+			FieldMappings: []QuickwitFieldMapping{
+				{
+					Name: "time",
+					Type: "datetime",
+					Fast: true,
+				},
+				{
+					Name: "uuid",
+					Type: "text",
+					Fast: true,
+				},
+				{
+					Name: "hostname",
+					Type: "text",
+					Fast: true,
+				},
+				{
+					Name: "priority",
+					Type: "text",
+					Fast: true,
+				},
+				{
+					Name: "source",
+					Type: "text",
+					Fast: true,
+				},
+				{
+					Name: "output",
+					Type: "text",
+				},
+				{
+					Name: "rule",
+					Type: "text",
+					Fast: true,
+				},
+				{
+					Name: "tags",
+					Type: "array<text>",
+					Fast: true,
+				},
+				{
+					Name: "output_fields",
+					Type: "json",
+					Fast: true,
+				},
+			},
+		},
+		SearchSettings: QuickwitSearchSettings{
+			DefaultSearchFields: []string{"rule", "source", "output", "priority", "hostname", "tags"},
+		},
+	}
+
+	if args.Config.Debug {
+		log.Printf("[DEBUG] : Quickwit - mapping: %v\n", mapping)
```

Comment:
```suggestion
		log.Printf("[DEBUG] : Quickwit - mapping: %#v\n", mapping)
```

Reply:
Done. Here's the result with docker:

<img width="1022" alt="Screenshot 2024-01-24 at 14 43 22" src="https://github.com/falcosecurity/falcosidekick/assets/12884347/a246e78f-af48-41ec-88d3-094752935bbb">